_tools/TicketDescriptions.py

```diff
@@ -81,30 +81,30 @@
 ntype_statements = ["Type: ",
                     "대응: ",
                     "Тип: ",
-                    ""]
+                    "类型"]
```

Comment:
Are you sure you don't want a colon here?

Reply:
lol, thank you.